### DISCUSSION TITLE
add Appimages builds

### DISCUSSION THREAD

**Discussion by eadmaster** [Murmele/Gittyup#390] [Ideas]
I prefer Appimages over flatpaks because they are easier to install.

Also mentioned [here](https://github.com/Murmele/Gittyup/issues/2).

**Comment by Murmele** [comment]
@eadmaster currently there is no plan for it. The problem is then we have to maintain another package. I think this has to be done by somebody externally. We will help of course if possible

**Comment by Murmele** [comment]
There is a pr pending, I have to finalize it and get it work on RHEL9
